src/compressor/zlib/ZlibCompressor.cc

```diff
@@ -187,6 +188,7 @@ int ZlibCompressor::compress(const bufferlist &in, bufferlist &out, boost::optio
 int ZlibCompressor::decompress(bufferlist::const_iterator &p, size_t compressed_size, bufferlist &out, boost::optional<int32_t> compressor_message)
 {
 #ifdef HAVE_QATZIP
+  dout(20) << "qat: " << qat_enabled << " " << __FUNCTION__ << dendl;
```

Comment:
debug message should be removed

Reply:
Thanks for your suggestion. It will be removed.